bindings/python/tests/test_api_responses.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Generic, TypeVar
+from json import load, loads, dumps
+from iota_sdk import RoutesResponse, CongestionResponse, ManaRewardsResponse, ValidatorResponse, ValidatorsResponse, CommitteeResponse, IssuanceBlockHeaderResponse, Block, OutputResponse, SlotCommitment
+
+
+base_path = '../../sdk/tests/types/api/fixtures/'
+T = TypeVar("T")
+
+
+def test_api_responses():
+    def test_api_response(cls_type: Generic[T], path: str):
```

Comment:
I think that's the first time I see some kind of generic in Python

Reply:
Probably because you don't really need them if you ignore the type checks :trollface: